### PR TITLE
Seeder fix

### PR DESCRIPTION
**Pull request by popov654**
Added seeders directory to the ORM config

### CODE REVIEW

File: src/mikro-orm.config.ts.example [hunk 1]
```diff
@@ -7,10 +7,13 @@ import { SeedManager } from '@mikro-orm/seeder';
 
 export default defineConfig({
   host: 'localhost',
-  port: 3307,
+  port: 3306,
```
Comment (B4nan) [inline src/mikro-orm.config.ts.example:10]: 3307 is correct

https://github.com/mikro-orm/nestjs-realworld-example-app/blob/master/docker-compose.yml#L9

**Review by B4nan — CHANGES_REQUESTED** [review]

**Comment by B4nan** [comment]
i would actually rather move the seeders folder, i don't recall why its in `database/seeders`, maybe that was some old default. it should be `src/seeders`, then it works without any config adjustments

File: src/mikro-orm.config.ts.example (hunk 1 above)
Comment (popov654) [reply src/mikro-orm.config.ts.example:10]: Sorry, I forgot that this setup is for dicker. I'm using a native MySQL local server

**Comment by popov654** [comment]
Yes, I agree it is better. Fixed

**Review by B4nan — APPROVED** [review]

**Pull request merged**